includes/class-wc-install.php

```diff
@@ -404,6 +404,17 @@ public static function needs_db_update() {
 		return ! is_null( $current_db_version ) && version_compare( $current_db_version, end( $update_versions ), '<' );
 	}
 
+	/**
+	 * See if we need the setup wizard or not.
+	 *
+	 * @since 4.6.0
+	 */
+	private static function maybe_enable_setup_wizard() {
```

Comment:
Nitpicky semantics: this function has changed a little bit and doesn't exactly enable the setup wizard, but sets a transient that might eventually be used to redirect to the onboarding.

Maybe we should change the name to something like `maybe_set_activation_transients()` so the next person looking at this doesn't mistake the usage.

Reply:
Thanks. Good call, updated!